task/cleanup-open-layers-map-buttons

michael-jaia opened the pull request.

michael-jaia committed:
Re-position + re-style ol map default buttons for better look on tablet

michael-jaia committed:
Clean-up attribution code + re-position ol credit for sleeker ui

ferrom-jaia committed:
Merge branch '1.y' into task/cleanup-ol-map-buttons

ferrom-jaia reviewed (changes requested):
We discussed the issue with the bottom right size indicator.

michael-jaia committed:
Remove fixed width from map scale div

Originally, I didn't realize it changed size with the zoom of the map.
Now the scale is back to freely adjusting based on the zoom.

ferrom-jaia committed:
Merge branch '1.y' into task/cleanup-ol-map-buttons

ferrom-jaia committed:
Merge branch '1.y' into task/cleanup-ol-map-buttons

ferrom-jaia reviewed (approved):
Looks good! Tested in sim!
